pkg/strategy/watch.go

```diff
@@ -14,7 +14,6 @@ import (
 )
 
 type watchResult struct {
-	ctx    context.Context
```

Comment:
This was a leftover change from trying to hunt down the memory leak. I decided to include it here.